apps/codeforafrica/src/lib/data/pagify/story.js

```diff
@@ -0,0 +1,16 @@
+import { getPost } from "@/codeforafrica/lib/data/utils/posts";
+
+async function story(api, context) {
```

Comment:
Functionally, is there any difference between this and opportunity? Can't they use the same function i.e. `post`?

Reply:
@kilemensi The only difference is that in `story` we have to include `Recent Stories`. I think I can set a different block for that.